test/e2e/node/kubelet_authz.go

```diff
@@ -112,7 +113,12 @@ func runKubeletAuthzTest(ctx context.Context, f *framework.Framework, endpoint,
 		},
 		true,
 	)
-	framework.ExpectNoError(err)
+
+	if err != nil && failOnAuthzUpdateErr {
+		framework.ExpectNoError(err)
+	} else if err != nil {
+		framework.Logf("Warning: Failed waiting for authorization: %v", err)
+	}
```

Comment:
This is ignoring all errors, not just time out errors.

Reply:
This is gone